components/function-controller/config/crd/bases/serverless.kyma-project.io_functions.yaml

```diff
@@ -1,273 +1,291 @@
+                  type: string
+                source:
+                  description: Source defines the source code of a function
+                  type: string
+                type:
+                  type: string
+              required:
+                - source
+              type: object
+            status:
+              description: 'FunctionStatus defines the observed state of FuncSONPath:
+              .status.phase'
+              properties:
+                baseDir:
+                  type: string
+                commit:
+                  type: string
+                conditions:
+                  items:
+                    properties:
+                      lastTransitionTime:
+                        format: date-time
+                        type: string
+                      message:
+                        type: string
+                      reason:
+                        type: string
+                      status:
+                        type: string
+                      type:
+                        description: ConditionType defines condition of function.
+                        type: string
+                    required:
+                      - status
+                    type: object
+                  type: array
+                reference:
+                  type: string
+                runtime:
+                  enum:
+                    - nodejs12
+                    - nodejs14
+                    - python38
+                  type: string
+                source:
+                  type: string
               type: object
-            runtime:
-              enum:
-              - nodejs12
-              - nodejs14
-              - python38
-              type: string
-            source:
-              description: Source defines the source code of a function
-              type: string
-            type:
-              type: string
-          required:
-          - source
-          type: object
-        status:
-          description: 'FunctionStatus defines the observed state of FuncSONPath:
-            .status.phase'
-          properties:
-            baseDir:
-              type: string
-            commit:
-              type: string
-            conditions:
-              items:
-                properties:
-                  lastTransitionTime:
-                    format: date-time
-                    type: string
-                  message:
-                    type: string
-                  reason:
-                    type: string
-                  status:
-                    type: string
-                  type:
-                    description: ConditionType defines condition of function.
-                    type: string
-                required:
-                - status
-                type: object
-              type: array
-            reference:
-              type: string
-            runtime:
-              enum:
-              - nodejs10
```

Comment:
I just noticed we used a temporary hack when deprecating `nodejs10` runtime and manually modified the generated CRD to keep the old enum value in Status object (it's needed because we allow existing nodejs10 functions to stay in cluster).

Could you please add this value manually for now? I'll open an issue to have this fixed so that no manual changes to CRD are needed in future.

Reply:
In both places, thanks!